### PR TITLE
Added tabs component. Based on MIT lisenced a11ytabs. Included origin…

### PR DESCRIPTION
**Pull request by oyessb**
…al copyright notice. Described modifications done as comments in top of files.

### CODE REVIEW

File: libs/pxweb2-ui/src/lib/components/Tabs/Tabs.module.scss [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+    background: var(--color-surface-action-subtle-hover, #dbebeb);
+  }
+
+  &:active {
+    background: var(--color-surface-action-subtle-active, #c3dcdc);
+  }
+
+  &:focus-visible {
+    border: 3px solid var(--color-border-focus-boxshadow, #162327);
+    outline: 3px solid var(--color-border-focus-outline, #7e4ef9);
```
Comment (SjurSutterudSagen) [inline libs/pxweb2-ui/src/lib/components/Tabs/Tabs.module.scss:39]: Same as line 2, missing prefix and should not have fallback.

File: libs/pxweb2-ui/src/lib/components/Tabs/Tabs.module.scss [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+    z-index: 1;
+  }
+}
+
+.underline {
+  position: absolute;
+  bottom: 0px;
+  height: 4px;
+  border-radius: var(--border-radius-full, 9999px);
+  background: var(--color-border-selected, #274247);
```
Comment (SjurSutterudSagen) [inline libs/pxweb2-ui/src/lib/components/Tabs/Tabs.module.scss:50]: Same as line 2, missing prefix and should not have fallback.

File: libs/pxweb2-ui/src/lib/components/Tabs/Tabs.module.scss [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+  width: 100%;
+  z-index: 1;
+}
+
+.label {
+  color: var(--color-text-default, #162327);
```
Comment (SjurSutterudSagen) [inline libs/pxweb2-ui/src/lib/components/Tabs/Tabs.module.scss:56]: Same as line 2, missing prefix and should not have fallback.

File: libs/pxweb2-ui/src/lib/components/Tabs/Tabs.module.scss [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+  /* Label/Medium */
```
Comment (SjurSutterudSagen) [inline libs/pxweb2-ui/src/lib/components/Tabs/Tabs.module.scss:57]: I might remember incorrectly here, but don't we have a file we can import this text style from?

File: libs/pxweb2-ui/src/lib/components/Tabs/Tabs.stories.tsx [hunk 5]
```diff
@@ -0,0 +1,118 @@
...
+        <Tab id="px-tab1" label="Tab1"></Tab>
+        <Tab id="px-tab2" label="Tab2"></Tab>
+        <Tab id="px-tab3" label="Tab3"></Tab>
+        <Tab id="px-tab4" label="Tab4"></Tab>
+      </Tabs>
+    </TabsProvider>
+  );
+};
+
+export const SrollableTabs: StoryFn<typeof Tabs> = () => {
```
Comment (SjurSutterudSagen) [inline libs/pxweb2-ui/src/lib/components/Tabs/Tabs.stories.tsx:30]: Is there a type in the name here? Should it be `ScrollableTabs`?

File: libs/pxweb2-ui/src/lib/components/Tabs/Tab.tsx [hunk 6]
```diff
@@ -0,0 +1,81 @@
+//Adapted from: Juliana Godoy Viana, a11ytabs (Original copyright notice included in copyright_notice.txt)
```
Comment (michaelpande) [inline libs/pxweb2-ui/src/lib/components/Tabs/Tab.tsx:1]: We should probably mention somewhere here that it is released with a MIT license

File: libs/pxweb2-ui/src/lib/components/Tabs/copyright_notice.txt [hunk 7]
```diff
@@ -0,0 +1,5 @@
+Copyright 2024 Juliana Godoy Viana
+
+Permission is hereby granted, free of charge, to any person obtaining a copy of this software and associated documentation files (the “Software”), to deal in the Software without restriction, including without limitation the rights to use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the Software, and to permit persons to whom the Software is furnished to do so, subject to the following conditions:
```
Comment (michaelpande) [inline libs/pxweb2-ui/src/lib/components/Tabs/copyright_notice.txt:3]: We should probably mention somewhere here that it is MIT license.

**Review by SjurSutterudSagen — APPROVED** [review]
LGTM now :)

**Pull request merged**